content/pt-br/docs/concepts/cluster-administration/system-logs.md

```diff
@@ -0,0 +1,135 @@
+---
+reviewers:
+- dims
+- 44past4
+title: Logs de Sistema
+content_type: concept
+weight: 60
+---
+
+<!-- overview -->
+
+Logs de componentes do sistema armazenam eventos que acontecem no cluster, sendo muito úteis para depuração. Seus níveis de detalhe podem ser ajustados para mais ou para menos. Podendo se ater, por exemplo, a mostrar apenas os erros que ocorrem no componente, ou chegando a mostrar cada passo de um evento. (Como acessos HTTP, mudanças no estado dos pods, ações dos controllers, ou decisões do scheduler)
+
+<!-- body -->
+
+## Klog
+
+[Klog](https://github.com/kubernetes/klog) é a biblioteca de logs do Kubernetes. Responsável por gerar as mensagens de log para os componentes do sistema. 
+
+Para mais informações acerca da sua configuração, veja a documentação da [ferramenta de linha de comando](https://kubernetes.io/docs/reference/command-line-tools-reference/) 
+
+Um exemplo do formato padrão dos logs da biblioteca:
+```
+I1025 00:15:15.525108       1 httplog.go:79] GET /api/v1/namespaces/kube-system/pods/metrics-server-v0.3.1-57c75779f-9p8wg: (1.512ms) 200 [pod_nanny/v0.0.0 (linux/amd64) kubernetes/$Format 10.56.1.19:51756]
+```
+
+### Logs Estruturados
+
+{{< feature-state for_k8s_version="v1.19" state="alpha" >}}
+
+{{< warning >}}
+A migração pro formato de logs estruturados é um processo em andamento. Nem todos os logs estão dessa forma na versão atual. Sendo assim, para realizar o parsing de arquivos de log, você também precisa lidar com logs não estruturados.
```

Comment:
Maybe we can change 'parsing' for something like 'tratamento/processamento', what you think? :)

Reply:
At first I did not think of a good word, so I chose to keep it as "parsing", since it's a common term among devs. But reading again now, "processamento" seems to be a nice fit. Modification done.